core/service/round.ts

```diff
@@ -202,143 +101,54 @@ class RoundService implements IRoundService {
         createTaskON(ActionTypes.ROUND_FINISH, roundEndTime);
     }
 
-    public async generateRound(timestamp: number = SlotService.getTime()): Promise<ResponseEntity<void>> {
-        /**
-         * if triggered by ROUND_FINISH event
-         */
-        if (
-            RoundRepository.getCurrentRound()
-        ) {
-            compose(
-                this.applyUnconfirmed,
-                this.sumRound
-            )(RoundRepository.getCurrentRound());
-
-            // store pound as previous
-            RoundRepository.setPrevRound(RoundRepository.getCurrentRound());
-            // TODO update prev round
-        }
+    public getMySlot(): number {
+        return RoundRepository.getCurrentRound().slots[this.keyPair.publicKey];
+    }
 
+    private distributeReward(round: Round, undo?: Boolean): void {
```

Comment:
i think, should be renamed to `processReward` or something. Because we can take back money with this method.